src/OWASPZAPDotNetAPI/NugetPackaging/OWASPZAPDotNetAPI.nuspec

```diff
@@ -2,17 +2,20 @@
 <package >
   <metadata>
     <id>OWASPZAPDotNetAPI</id>
-    <version>1.0.0</version>
+    <version>2.6.0</version>

```

Comment:
I mean if there's a bug in this code, you'd have to reissue the dot net api as 2.6.1 or whatever, ending up outo of sync on version numbers...